src/views/DatabaseProfilerView.vue

```diff
@@ -95,7 +95,7 @@ SPDX-License-Identifier: AGPL-3.0-or-later
 									<div class="detail-see">
 										See {{ similarQueries[index].singles.length }} in:
 										<a v-for="(simIndex, i) in similarQueries[index].singles" :key="i" :href="anchor(simIndex)">
-											{{ simIndex }}&nbsp;
+											{{ simIndex }}&#32;
```

Comment:
You want a non breaking space but Vue only adds a space or what is the problem?
Because it will always convert whitespace to spaces here by the compiler.
If you really need non breaking spaces then you need to use template interpolation like:

```suggestion
											{{ simIndex }}{{'\xa0'}}
```

Or in one:

```suggestion
											{{ simIndex + '\xa0' }}
```

Reply:
But probably its even cleaner to remove the space completely and just add some margin to the `a` element?